inc/migration.class.php

```diff
@@ -403,7 +403,7 @@ function addField($table, $field, $type, $options = []) {
             $this->change[$table][] = "ADD `$field` $format ".$params['comment'] ." ".
                                       $params['null'].$params['first'].$params['after'];
 
-            if (!empty($params['update'])) {
+            if ($params['update'] !== '') {
```

Comment:
Needed as we often have `update => 0` (three times in this file for some new entity setting) and `empty(0)` is true.

Reply:
Is it possible to add a unit test on that case?

Reply:
Done in 43d1627.